web-module/src/main/webapp/resources/structEditJS/dateTime/celements-date-time-field.js

```diff
@@ -319,7 +316,10 @@
 
     #onDateTimeChange() {
       this.#updateComponentValuesFromInput();
-      this.#setMinMax(this.#dateTimeComponent.siblings);
+      // let my siblings know that I have changed
+      this.#dateTimeComponent.siblings

```

Comment:
why not using Oberver-Pattern? you could use CELEMENTS.mixins.Observable (see e.g. `class CelOverlay` in CelOverlay.js) This would make it more generic and safe, moreover any class interessted could register for the event.